src/tests/compiletest.md

```diff
@@ -501,23 +501,22 @@ To run the tests in a different mode, you need to pass the `--compare-mode`
 CLI flag:
 
 ```bash
-./x.py test src/test/ui --compare-mode=nll
+./x.py test src/test/ui --compare-mode=chalk
 ```
 
 The possible compare modes are:
 
-* `nll` — Runs in the "true" NLL mode with `-Zborrowck=mir`.
-  See [UI compare modes](ui.md#compare-modes) for more.
-* `polonius` — Runs with Polonius with `-Zpolonius -Zborrowck=mir`, and reuses
-  the `nll` stderr files.
+* `polonius` — Runs with Polonius with `-Zpolonius -Zborrowck=mir`.
```

Comment:
No `-Zborrowck`

Reply:
It looks like it is still defined [here](https://github.com/rust-lang/rust/blob/09d52bc5d4260bac8b9a2ea8ac7a07c5c72906f1/src/tools/compiletest/src/runtest.rs#L1942). I'm guessing that should be removed, too?

Reply:
Oh. Yeah, that's gonna fail.

Reply:
OK, posted https://github.com/rust-lang/rust/pull/97894 and https://github.com/rust-lang/rustc-dev-guide/pull/1367 with a fix.